src/tokens/solana.tokenlist.json

```diff
@@ -81724,4 +81738,4 @@
     "minor": 3,
     "patch": 3
   }
-}
\ No newline at end of file
+}
```

Comment:
Hey mate I don't think it's happy with the new line here.
You will probably need to change that config setting in your editor that adds a new line at the end of the file.
Then close this PR and create a new one and the auto-merger should pick it up fine :)